documentation/8260-Dylan-WhatsNewEncouragedUpdateDocumentation (#8262)
department-of-veterans-affairs/va-mobile-app: VAMobile/documentation/docs/App Features/WhatsNew/WhatsNew.md

> # Whats New

Reviewer: ```suggestion
# What's New
```

Author: added

## Feature Summary

> Whats new displays to the veteran what is new in the version that they upgraded to. This alert only shows if the the version of the app the user is on is the most recent in the store and if there is an important enough change to annouce with the provided notes to display to the user. The user can dismiss this alert from reappearing for the current version.

Reviewer: ```suggestion
What's new displays to the veteran what is new in the version that they upgraded to. This alert is only shown if the version of the app the user is on is the most recent in the store and if there is asignificant change to announce with the provided notes to display to the user. The user can dismiss this alert from reappearing for the current version.
```

Author: added

## Use Cases

> * Use Case 1: The user is on the most recent version available in the store and there is notes for the alert to display resulting in the alert displaying(see screenshot)

Reviewer: ```suggestion
* Use Case 1: The user is on the most recent version available in the store, and there are notes for the alert to display, resulting in the alert being shown (see screenshot)
```

Author: added

> * Use Case 2: The user is on the most recent version with notes to display, but the user has dismissed the alert causing it to no longer be displayed

Reviewer: ```suggestion
* Use Case 2: The user is on the most recent version with notes to display, but the user has dismissed the alert, causing it to no longer be displayed
```

Author: added

> * Use Case 3: The user is on a older version of the app and instead sees [Encouraged Update](../EncouragedUpdate/EncouragedUpdate.md)

Reviewer: ```suggestion
* Use Case 3: The user is on an older version of the app and instead sees [Encouraged Update](../EncouragedUpdate/EncouragedUpdate.md)
```

Author: added

Reviewer: @Sparowhawk Looks like this suggestion wasn't applied

> * Use Case 3: The user is on the most recent version but there are no notes to be displayed so there is no alert

Reviewer: ```suggestion
* Use Case 3: The user is on the most recent version, but there are no notes to be displayed, so there is no alert
```

Author: added

> * Use Case 3: The user is on the most recent version, but there are no notes to be displayed, so there is no alert

Reviewer 2: ```suggestion
* Use Case 4: The user is on the most recent version, but there are no notes to be displayed, so there is no alert
```
